### PR TITLE
[LTR] Implement Eomer, Marshal of Rohan

### PR DESCRIPTION
**Pull request by Susucre**
Also, small refactor adding EachTurn to the name of both `setTriggersOnce` and `setDoOnlyOnce`.

### CODE REVIEW

**Comment by Susucre** [comment]
[[Eomer, Marshal of Rohan]]

File: Mage.Sets/src/mage/cards/e/EomerMarshalOfRohan.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        this.subtype.add(SubType.KNIGHT);
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(4);
+
+        // Haste
+        this.addAbility(HasteAbility.getInstance());
+
+        // Whenever one or more other attacking legendary creatures you control die, untap all creatures you control. After this phase, there is an additional combat phase. This ability triggers only once each turn.
+        Ability ability = new DiesCreatureTriggeredAbility(
+            new UntapAllEffect(StaticFilters.FILTER_CONTROLLED_CREATURE), false, filter
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/e/EomerMarshalOfRohan.java:50]: FILTER_CONTROLLED_CREATURES

File: Mage/src/main/java/mage/abilities/TriggeredAbilityImpl.java [hunk 2]
```diff
@@ -120,9 +120,9 @@ public boolean checkUsedAlready(Game game) {
         return lastTurnUsed != null && lastTurnUsed == game.getTurnNum();
     }
 
-    public TriggeredAbility setDoOnlyOnce(boolean doOnlyOnce) {
+    public TriggeredAbility setDoOnlyOnceEachTurn(boolean doOnlyOnce) {
```
Comment (xenohedron) [inline Mage/src/main/java/mage/abilities/TriggeredAbilityImpl.java:123]: Interesting that this one is defined only here in the Impl and the other is defined in the Interface

File: Mage/src/main/java/mage/abilities/TriggeredAbilityImpl.java [hunk 3]
```diff
@@ -104,14 +104,14 @@ public boolean checkTriggeredAlready(Game game) {
         return lastTurnTriggered == null || lastTurnTriggered != game.getTurnNum();
     }
 
-    public TriggeredAbility setTriggersOnce(boolean triggersOnce) {
```
Comment (xenohedron) [inline Mage/src/main/java/mage/abilities/TriggeredAbilityImpl.java:107]: should this be annotated with `@Override`?

**Review by xenohedron — APPROVED** [review]

**Pull request merged**